.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!--- Thank you for opening a pull request! Here are some helpful tips:
+     
+      1. To solicit reviewers: 
+           the @usrse-maintainers will automatically notify those offering to review.
```

Comment:
Do you mean to ask people to "tag" `@usrse-maintainers` here? Maybe change to:
> `Tag the @usrse-maintainers to automatically notify those offering to review.`

Reply:
I don't ask them to tag because @USRSE/usrse-maintainers is already in the template. I'll make that more clear in the line there!